### PR TITLE
Add some common used operators' conversion

### LINKED ISSUE #39: Add some common used operators' conversion

### PR DESCRIPTION
**Pull request by kuke**
Resolve #39

### CODE REVIEW

**Comment by varunarora** [comment]
Super glad you just independently worked on these rather than waiting for a model where they show up

File: fluid_onnx/ops.py [hunk 1]
```diff
@@ -195,20 +183,10 @@ def dropout_op(operator, block):
...
-    return make_node(
-        'Add',
-        inputs=inputs['X'] + inputs['Y'],
-        outputs=outputs['Out'],
-        axis=attrs['axis'],
-        broadcast=1)
-
-
-def elementwise_mul_op(operator, block):
+def elementwise_ops(op_type, operator, block):
```
Comment (varunarora) [inline fluid_onnx/ops.py:186]: Do we have broadcasting control in Fluid? I don't remember the API

File: fluid_onnx/ops.py [hunk 2]
```diff
@@ -459,8 +421,29 @@ def randomuniformlike_op():
     pass
 
 
-def reciprocal_op():
-    pass
+def reduce_ops(op_type, operator, block):
```
Comment (varunarora) [inline fluid_onnx/ops.py:424]: A good explainer comment on these bigger op functions (things that support more than one fluid op) would be awesome

File: tests/test_reduce_ops.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        self.inputs = {'X': np.random.random((5, 6, 7, 8)).astype("float32")}
+        self.attrs = {
+            'dim': 2,
+            'keep_dim': self.keep_dim,
+            'reduce_all': self.reduce_all
+        }
+        self.outputs = {'Out': np.zeros((1, 1))}
+
+    def init_op_type(self):
+        self.op_type = "reduce_sum"
```
Comment (varunarora) [inline tests/test_reduce_ops.py:34]: Small thing, but just be consistent whether you use single or double quotes. Small formatting thing

**Review by varunarora — APPROVED** [review]
Small things and then ready to go :+1:

File: fluid_onnx/ops.py (hunk 1 above)
Comment (kuke) [reply fluid_onnx/ops.py:186]: No. The API cannot control broadcast out of the operators. Please take a look [here](https://github.com/PaddlePaddle/Paddle/blob/c26204027fa30c5925ae3e169f70b0197674a86b/paddle/fluid/operators/elementwise_op.h#L55)

File: fluid_onnx/ops.py (hunk 2 above)
Comment (kuke) [reply fluid_onnx/ops.py:424]: Good suggestion. I will add some comments to these functions in next PR.

File: tests/test_reduce_ops.py (hunk 3 above)
Comment (kuke) [reply tests/test_reduce_ops.py:34]: OK. I will unify the usage of quotes.

**Review by kuke — COMMENTED** [review]
@varunarora Thanks! There are no more available models to cover most of the rest operators, so I think we'd better begin to implement their conversion one by one, and I will add more operators' conversion. Since we are on the Labor Day holiday, next PR may not come very soon.

**Pull request merged**